### DISCUSSION TITLE
Display all fields for users by default

### DISCUSSION THREAD

**Discussion by RobinFrcd** [directus/directus#13047] [Archive]
Hi,
I have a few tables with a lot of fields, and I still want all my users to display them by default. 
In the admin panel, the fields are not marked as hidden, they just don't show up in the UI by default. Is there a way to change this ?

For now, the only solution I found is to set the layout I want on the admin account and then run this SQL query:

```sql
update
	directus_presets
set
	layout_query = base_layout.layout_query
from
	(
	select
		layout_query,
		collection
	from
		directus_presets dp
	where
		dp."user" = '3ddef88a-244b-4c22-a8d7-e87943985c61' -- admin user
         ) base_layout
where
	directus_presets.collection = base_layout.collection;
```

**Comment by rijkvanzanten** [comment]
You can go to settings > presets and change the scope for that particular preset from your user to Global to make it the global defaults for everybody 👍🏻

**Reply by RobinFrcd** [reply]
That's indeed much cleaner than messing with the DB, thanks! :+1:
